onnxruntime/contrib_ops/cuda/math/bias_softmax_impl.cu

```diff
@@ -215,7 +215,7 @@ Status BiasSoftmaxImpl(cudaStream_t stream, cudnnHandle_t cudnn_handle, T* outpu
                         static_cast<size_t>(batch_count * element_count));
 
   // invoke cuda DNN library for Y = softmax(X)
-  std::vector<int64_t> dims({batch_count, 1, 1, element_count});
+  const int64_t dims[]{batch_count, 1, 1, element_count};
```

Comment:
>dims [](http://example.com/codeflow?start=16&length=4)

curious what is the intention for this change.

Reply:
Get rid of std::vector according to the coding standard in Dimitri's email.